src/components/Footer.tsx

```diff
@@ -1,5 +1,47 @@
+import Image from "next/image";
+import Link from "next/link";
+import logo from "@/../public/logo.webp";
+
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div className="bg-naama-blue-200 text-naama-ivory-100 font-nunito w-full items-center justify-center border p-10">
```

Comment:
```suggestion
    <div className="bg-naama-blue-200 text-naama-ivory-100 w-full items-center justify-center border p-10">
```

directly calling this seems to make the font not appear, not sure why exactly but remove it for now, should work